tuner/pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [project]
-name = "SHARK Tuner"
+name = "SHARK-Tuner"
```

Comment:
Why?

Reply:
The space was causing my pip install of the tuner to fail for some reason, so I just made it a dash. I forgot to remove this change before pushing. I'm not sure why it fails to pip install on my machine, but if there is no preference for having a space then I think it's probably safer to use a dash.

Reply:
Oh I've never tried installing it, I don't know if it actually works. I mostly copy-pasted from other project file in this repo